### PR TITLE
add: max option to avoid more chars then the max.

### PR DESCRIPTION
**Pull request by wbotelhos**
To use `maxlength` from blocks the backspace and so.

### CODE REVIEW

**Comment by plentz** [comment]
@wbotelhos sorry for taking so long to reply =~

what browser needs this? I've just tested with chrome(last stable) and using maxlength works as expected :)

File: src/jquery.maskMoney.js [hunk 1]
```diff
@@ -124,7 +125,15 @@
...
+                function withinMaximum() {
+                    return $input.val().match(/\d/g).length < settings.max;
+                }
+
                 function canInputMoreNumbers() {
+                    if (settings.max && !withinMaximum()) {
+                        return false;
+                    }
+
                     var haventReachedMaxLength = !($input.val().length >= $input.attr("maxlength") && $input.attr("maxlength") >= 0),
```
Comment (plentz) [inline src/jquery.maskMoney.js]: this is the code that should handle this :)

**Comment by plentz** [comment]
#157

**Comment by wbotelhos** [comment]
I found the problem: `prefix`.
I must to count it on `maxlength`

**Pull request closed without merging**